src/main/java/uk/gov/hmcts/reform/bulkscan/Application.java

```diff
@@ -1,10 +1,17 @@
 package uk.gov.hmcts.reform.bulkscan;
 
 import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.SpringBootConfiguration;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.cloud.openfeign.EnableFeignClients;
+import org.springframework.context.annotation.ComponentScan;
 
-@SpringBootApplication
-@SuppressWarnings("HideUtilityClassConstructor") // Spring needs a constructor, its not a utility class
+@EnableFeignClients(basePackages = {"uk.gov.hmcts.reform.bulkscan.*", "uk.gov.hmcts.reform.bulkscan.services"})
```

Comment:
you don't need uk.gov.hmcts.reform.bulkscan.services if you are putting uk.gov.hmcts.reform.bulkscan.*. I believe * will cover entire parent directory

Reply:
make sense. removed it